Tasksets/sets.md

```diff
@@ -0,0 +1,9 @@
+# Sets of tasks
```

Comment:
Do we want to have this separate or in the normal Readme?

Reply:
Don't know really. I needed to have this available for tests, I just created a random file. I usualy use lots of .md files to get detailled information, and keep the Readme to simple and fast information with links to the details inside. This can be done as you see fit.

Reply:
Both is fine by me.
I'm a fan of single file readmes simply because it makes searching easier.
On the other hand I also totally failed to update the Readme with all the new features :D

Let's just leave it there for the moment.